### ISSUE TITLE
2019-6-5

### ISSUE THREAD

**Issue by ssfc** [ssfc/wangdao#24]
731.2.0 - Problem statement: SIGNS Dataset
732.2.1 - Create placeholders
733.Create Place holders
734.2.2 - Initializing the parameters
735.Initialize Parameters
736.2.3 - Forward propagation in tensorflow
737.Forward Propagation
738.2.4 Compute cost
739.Compute Cost - softmax cross entropy
